vehicle-python/src/vehicle_lang/compile/python.py

```diff
@@ -59,14 +67,63 @@
     "load_loss_function",
 ]
 
+
+################################################################################
+## Variable domain
+################################################################################
+
+
+class VariableDomain(AbstractVariableDomain[np.ndarray]):
```

Comment:
We really shouldn't be mentioning numpy in the standard Python compilation. This should once more just be `Vector`.